components/admin-panel/sections/ReceiptTemplateForm.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import { Flex } from '../../Grid';
+import PreviewModal from '../../PreviewModal';
+import StyledButton from '../../StyledButton';
+import StyledInput from '../../StyledInput';
+import StyledTextarea from '../../StyledTextarea';
+import { Label, P, Span } from '../../Text';
+
+const ReceiptTemplateForm = ({
+  defaultTemplate,
+  defaultReceiptTitle,
+  receiptInfo,
+  receiptTitlePlaceholder,
+  receiptInfoPlaceholder,
+  onChangeReceiptTitle,
+  onChangeReceiptInfo,
+}) => {
+  const [showPreview, setShowPreview] = React.useState(false);
+
+  return (
+    <React.Fragment>
+      <Label htmlFor="receipt-title" color="black.800" fontSize="16px" fontWeight={700} lineHeight="24px">
+        <FormattedMessage defaultMessage="Receipt title" />
+      </Label>
+      <StyledInput
+        inputId="receipt-title"
```

Comment:
Shouldn't that be `id`? I believe `inputId` is for `react-select`. Same for the other input below. You can try by clicking on the label, it should focus the input

```suggestion
        id="receipt-title"
```

Reply:
Ops, you are correct. I've changed this. 👍🏽